app/Main.hs

```diff
@@ -22,16 +23,18 @@ main = do
     let configFile = case args of
             [cnf] -> cnf
             _ -> error usage
-    run configFile repostUpdates
+    run configFile
   where
```

Comment:
The basic problem that I had to do a lot of unrelated changes otherwise. It's not possible to use two FileCache layer in one monad stack, because they have different types. As a result MonadStack should be reordered as:
```
runGitterT $ flip runReaderT config $ do
    runDiscourceT $ runFileCacheT ...
    runRssT $ runFileCacheT ..
```
so but it will require changes in how we pass arguments to `run`, and other changes in existing instances.